nemo-physical/src/tabular/operations/filter.rs

```diff
@@ -125,7 +125,8 @@ impl GeneratorFilter {
             }
         }
 
-        unreachable!("Filter must only use markers from the table.")
+        // Compute constant columns in the first column
+        table[0]
```

Comment:
From my high level understanding: Would it not be more intuitive to make this function return an `Option`?
Because in an example where the filter is only using constants, the column marker should really be `none`, no? (I could be wrong here, since I'm not totally sure what the `OperationColumnMarker` is.)

Reply:
I guess the way the function is named, it makes sense. I changed it accordingly and set the default value after the call.